Week1/numberRounding.js

```diff
@@ -0,0 +1,10 @@
+//round and compair numbers
+let z = 7.25;
+console.log(z);
+let a = Math.round(z);
+console.log(a);
+let numbers = [a, z];
+numbers.sort(function(a, b) { return b - a; });
```

Comment:
haha wow, cool solution!

That's really nice, there is an easier solution though (but not as cool): [Math.max](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max)

Reply:
I prefer your way. Math.max. updated